hopsanc/src/hopsanc.cpp

```diff
@@ -115,6 +111,24 @@ int getDataVector(const char* variable, double *data)
         }
     }
 
+    //Check for alias if splitVar is of size one
+    if(splitVar.size() == 1 && pSystem->getAliasHandler().hasAlias(splitVar[0])) {
+        hopsan::HString compName, portName;
+        int varId;
+        pSystem->getAliasHandler().getVariableFromAlias(splitVar[0], compName, portName, varId);
+        hopsan::Component *pComp = pSystem->getSubComponent(compName);
+        hopsan::Port *pPort = pComp->getPort(portName);
+        std::vector< std::vector<double> > *pLogData = pPort->getLogDataVectorPtr();
+        for (size_t t=0; t<spCoreComponentSystem->getNumActuallyLoggedSamples(); ++t) {
```

Comment:
Maybe this should be pSystem, in case the number of logged samples would be different from the main system.
Not sure if that can happen.

Reply:
The hopsanc interface can only set number of log samples for the entire model. Is it even possible to have different number of samples in a sub system? Didn't we take away that feature, or was that only from the gui?

Reply:
But still, since we take the logdata vector from a pPort in a component in pSystem, it makes sens to ask pSystem for the actual number of log samples. Even though they should be the same as the main system.

Reply:
That's true. I changed it now.